client/db/bolt/db_test.go

```diff
@@ -59,17 +56,7 @@ func newTestDB(t *testing.T) (*BoltDB, func()) {
 
 func TestMain(m *testing.M) {
 	defer os.Stdout.Sync()
-	doIt := func() int {
-		var err error
-		tDir, err = os.MkdirTemp("", "dbtest")
-		if err != nil {
-			fmt.Printf("error creating temporary directory: %v\n", err)
-			return -1
-		}
-		defer os.RemoveAll(tDir)
-		return m.Run()
-	}
-	os.Exit(doIt())
+	os.Exit(m.Run())
 }
```

Comment:
I think you can just remove `TestMain` now. I'm unsure of what `os.Stdout.Sync()` accomplishes... Or you can leave it I guess, don't think it hurts anything.